Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -287,13 +303,37 @@ public extension Attribute where Context == HTML.InputContext {
     /// Assign whether the element is required before submitting the form.
     /// - parameter isRequired: Whether the element is required.
     static func required(_ isRequired: Bool) -> Attribute {
-        isRequired ? Attribute(name: "required", value: "true") : .empty
+        isRequired ? Attribute(name: "required", value: nil, ignoreIfValueIsEmpty: false) : .empty
     }
     
     /// Assign whether the element should be autofocused when the page loads.
     /// - parameter isOn: Whether autofocus should be turned on.
     static func autofocus(_ isOn: Bool) -> Attribute {
-        isOn ? Attribute(name: "autofocus", value: "true") : .empty
+        isOn ? Attribute(name: "autofocus", value: nil, ignoreIfValueIsEmpty: false) : .empty
+    }
+
+    /// Assign whether the element should be read-only.
+    /// - parameter isReadonly: Whether the input is read-only.
+    static func readonly(_ isReadonly: Bool) -> Attribute {
+        isReadonly ? Attribute(name: "readonly", value: nil, ignoreIfValueIsEmpty: false) : .empty
+    }
+
+    /// Assign whether the element should be disabled.
+    /// - parameter isDisabled: Whether the input is disabled.
+    static func disabled(_ isDisabled: Bool) -> Attribute {
+        isDisabled ? Attribute(name: "disabled", value: nil, ignoreIfValueIsEmpty: false) : .empty
+    }
+
+    /// Assign whether the element should allow the selection of multiple values.
+    /// - parameter isMultiple: Whether multiple values are allowed.
+    static func multiple(_ isMultiple: Bool) -> Attribute {
```

Comment:
`isMultiple` reads a bit awkwardly here, IMO, since it's not the element itself that is or isn't "multiple". How about simply naming this parameter `isOn` or `isEnabled`, similar to how some of the other attribute parameters are named?

Reply:
Fixed in e1ac11c668e9b993e81f05326530f746d7342c74